src/Parser/Rogue/Subtlety/Modules/Features/Abilities.js

```diff
@@ -10,26 +11,90 @@ class Abilities extends CoreAbilities {
       spell: SPELLS.VANISH,
       category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
       getCooldown: haste => 90,
-      recommendedCastEfficiency: 1.0,
+      //Vanish is often delayed.
+      recommendedCastEfficiency: 0.90,
     },
     {
       spell: SPELLS.SHADOW_BLADES, // TODO: Reduced by Convergence of Fates
       category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
       getCooldown: haste => 180,
       recommendedCastEfficiency: 1.0,
+      extraSuggestion: "In most cases should be used on cooldown", 
     },
     {
       spell: SPELLS.SYMBOLS_OF_DEATH,
-      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-      getCooldown: haste => 30, // TODO: Add T20 4Sets, reduce cd by 5 sec.
+      category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+      getCooldown: (haste, combatant) => (30 - (combatant.hasBuff(SPELLS.SUB_ROGUE_T20_2SET_BONUS.id) ? 5 : 0)), 
       recommendedCastEfficiency: 0.95,
+      importance: ISSUE_IMPORTANCE.MAJOR,
+      extraSuggestion: "This is the most important rotational ability, try to always use it on cooldown.",      
     },
     {
       spell: SPELLS.SHADOW_DANCE,
-      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-      getCooldown: haste => 60, // TODO: Reduced by a passive.
+      category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+      getCooldown: haste => 60, // TODO: Reduced by a passive
       charges: 2,
-      recommendedCastEfficiency: 0.95,
+      noSuggestion: true,
+    },
+    {
+      spell: SPELLS.GOREMAWS_BITE,
+      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown: haste => 60,
+      recommendedCastEfficiency: 0.9,
+    },   
```

Comment:
You don't want any extraSuggestion here to aid the player in how this should be used? afaik it's generally used in combination with other stuff aswell, ye?

Reply:
The exact condition again depends on talents but I think I found suitable wording for it